docs/serverless/README.md

```diff
@@ -1,52 +0,0 @@
-# Serverless Agent
```

Comment:
How is this change related to the rest of the PR? Also, `docs/serverless` if kept should be owned by serverless, not Agent Platform

Reply:
Yeah I'm going to remove this change, it's not related to the rest of the PR. Sorry for the review spam.